stack/.github/workflows/push-image.yml

```
@@ -35,6 +35,24 @@ jobs:
         output: "/github/workspace/run.oci"
         token: ${{ secrets.PAKETO_BOT_GITHUB_TOKEN }}
 
+    - name: Get Registry Repo Name
+      id: registry-repo
+      run: |
+        # Strip off the Github org prefix and 'stack' suffix from repo name
+        # paketo-buildpacks/some-name-stack --> some-name
+        echo "::set-output name=name::$(echo "${{ github.repository }}" | sed 's/^.*\///' | sed 's/\-stack$//')"
+
+    - name: Setup gcloud
+      if: ${{ steps.registry-repo.outputs.name == "bionic"-* }}
+      uses: google-github-actions/setup-gcloud@master
+      with:
+        version: '270.0.0'
+        service_account_key: ${{ secrets.GCR_PUSH_BOT_JSON_KEY }}
+
+    - name: Configure docker gcloud auth
+      if: ${{ steps.registry-repo.outputs.name == "bionic"-* }}
+      run: gcloud auth configure-docker
```

Taking a look at the documentation for `gcloud auth configure-docker`, I'm seeing an example that makes me wonder if this should be
```suggestion
      run: gcloud auth configure-docker gcr.io
```
Have you tried running either of these locally / in CI somewhere?

---

@fg-j I did run these steps locally without the `gcr.io`. I took the steps directly from [the old workflows we were using](https://github.com/paketo-buildpacks/full-stack-release/blob/0df1a723056d98cc1c465e78f225b7dcde493121/.github/workflows/create-release.yml#L26-L32). It can't hurt to specify though